apps/src/gamelab/blocks.js

```diff
@@ -15,6 +15,21 @@ let sprites = () => {
   });
 };
 
+const limitedColours = [
+  '#000000', // BLACK
+  '#808080', // GREY
+  '#c3b091', // KHAKI
+  '#ffffff', // WHITE
+  '#ff0000', // RED
+  '#ff77ff', // PINK
+  '#ffa000', // ORANGE
+  '#ffff00', // YELLOW
+  '#228b22', // GREEN
+  '#0000cd', // BLUE
+  '#7fffd4', // AQUAMARINE
+  '#843179', // PLUM
```

Comment:
Should these be red/yellow/blue/orange/green/purple?

> only fully saturated primary and secondary colors

Reply:
ooh, yes, I was definitely using the wrong colors! I'm not sure what secondary colors are in this context but I'd assume cyan / magenta not orange / purple:
<img width="378" alt="screen shot 2018-11-26 at 1 44 44 pm" src="https://user-images.githubusercontent.com/8001765/49044024-7ae98680-f181-11e8-80b4-ff724f41aaf4.png">

@ryansloan can you please confirm the colors in the left sidebar of the above image look correct?

Reply:
Ryan says yes.